File: game-of-go-spring-boot/src/main/java/io/github/aylesw/igo/game/GameInfo.java
package io.github.aylesw.igo.game;

import io.github.aylesw.igo.dto.PlayerDto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class GameInfo {
    String gameId;
    PlayerDto blackPlayer;
    PlayerDto whitePlayer;
    GameConfig gameConfig;
    GameState gameState;
    GameResult gameResult;
}

File: game-of-go-angular/src/app/common/game-info.ts
import { GameConfig } from "./game-config";
import { GameResult } from "./game-result";
import { GameState } from "./game-state";
import { Player } from "./player";

export class GameInfo {
    gameId?: string;
    blackPlayer?: Player;
    whitePlayer?: Player;
    gameConfig?: GameConfig;
    gameState?: GameState;
    gameResult?: GameResult;
}
